kibana/README.md

```diff
@@ -996,3 +996,64 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
+## Impersonation
+
+User with specific roles (`admin` or `unrestricted`) and permissions (`impersonation` section in the `readonlyrest.yml` file)
+can impersonate (imitate behaviors or actions of the user) another user.
+
+### Test settings
```

Comment:
For me, it'd be nice to have a section where we describe what the Impersonation feature looks like.  I mean, that we could tell a user/admin how to configure sth (like test settings), but first they should be aware of what is this. And why they even have to do it.